src/recipeHydrator.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace GRUB;
+class RecipeHydrator
+{
+    private $recipes=[];
```

Comment:
```suggestion
    private $recipes = [];
```
the rest of the code base have spaces around the = sign, so I'd do that too for consistency